tests/components/tessie/test_lock.py

```diff
@@ -72,18 +83,39 @@ async def test_locks(
         assert hass.states.get(entity_id).state == STATE_UNLOCKED
         mock_run.assert_called_once()
 
+
+async def test_speed_limit_lock(
```

Comment:
Could we just add to test that the issues are created correctly?

Reply:
Done. Sorry for the delay.